src/org/omegat/gui/preferences/view/AppearanceController.java

```diff
@@ -55,21 +67,32 @@ public String toString() {
     }
 
     private void initGui() {
+        UIDesignManager.getThemes().forEach(themeSelectionModel::addElement);
         panel = new AppearancePreferencesPanel();
         // TODO: Properly abstract the restore function
         panel.restoreWindowButton
                 .addActionListener(e -> MainWindowUI.resetDesktopLayout((MainWindow) Core.getMainWindow()));
     }
-
+    
     @Override
     protected void initFromPrefs() {
+        String currentTheme = Preferences.getPreferenceDefault(Preferences.THEME_SELECTED_NAME, "Default");
```

Comment:
We use the magic string `Default` in a lot of places. I would prefer a constant in `Preferences.java`.

Reply:
done.